internal/pkg/grpcsupport/grpcservicegen.go

```diff
@@ -0,0 +1,350 @@
+package grpcsupport
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/TIBCOSoftware/flogo-lib/logger"
+)
+
+const (
+	serviceName = "\nservice "
+	grpcGenPath = "github.com/TIBCOSoftware/mashling/gen/grpc"
+)
+
+//Logger
+var log = logger.GetLogger("mashling-cli-grpc-gen")
+
+var (
+	protoPath     string
+	protoFileName string
+	protoImpPath  string
+	goPath        string
+	cmdExePath    string
+	appPath       string
+)
+
+//MethodInfoTree holds method information
+type MethodInfoTree struct {
+	MethodName    string
+	MethodReqName string
+	MethodResName string
+	serviceName   string
+}
+
+//ProtoData holds proto file data
+type ProtoData struct {
+	Timestamp      time.Time
+	MethodInfo     []MethodInfoTree
+	ProtoImpPath   string
+	RegServiceName string
+	ProtoName      string
+}
+
+func assignValues() {
```

Comment:
Instead of pushing the generated code right into the primary Mashling repo maybe we should take the current approach used by the "create" command in the CLI? Unpack the core Mashling gateway code, put the generated source there, and then directly build/compile the contents? This keeps each build separate and also automates another step in the gRPC generation process I think. We can skip the 'dep ensure' step so it should be pretty quick.